index.html

```diff
@@ -20,58 +19,61 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
-    <nav class="navigation">
-        <img class="logo" src="img/karma-logo.svg" alt="Karma logo image" width="26" />
-        <ul class="links">
-            <li class="bold">Meet Karma</li>
-            <li>How it Works</li>
-            <li>Store</li>
-            <li>Blog</li>
-            <li>Help</li>
-            <li>Login</li>
-        </ul>
-    </nav>
+    <header>
+        <div class="upper-section">
+            <img class="logo" src="img/karma-logo.svg" alt="Karma logo image" width="26" />
+            <nav class="navigation">
+                <ul class="links">
+                    <li class="bold">Meet Karma</li>
+                    <li>How it Works</li>
+                    <li>Store</li>
+                    <li>Blog</li>
+                    <li>Help</li>
+                    <li>Login</li>
+                </ul>
+            </nav>
+        </div>
+    </header>
 
     <main>
         <section class="introduction">
             <div class="description">
                 <h1>Introducing Karma</h1>
                 <h3>Bring WiFi with you, everywhere you go.</h3>
-            </div>
-            <div class="button">
-                <button>Learn More</button>
+                <button class="button"><p>Learn More</p>
+                </button>
             </div>
         </section>
 
         <section class="boxes">
-            <h2>Everyone needs a little Karma.</h2>
-            <ul class="features">
-                <li><img width="127" height="127" src="img/icon-devices.svg" alt="electronic devices image" /></li>
-                <div class="caption">
-                    <h3>Internet for all devices</h3>
-                </div>
-                <li><img width="127" height="127" src="img/icon-coffee.svg" alt="cup of coffee image" /></li>
-                <div class="caption">
-                    <h3>Boost your productivity</h3>
-                </div>
-                <li><img width="127" height="127" src="img/icon-refill.svg" alt="gas refilling device image" /></li>
-                <div class="caption">
-                    <h3>Pay as You Go</h3>
-                </div>
-            </ul>
+            <div class="content">
+                <h2>Everyone needs a little Karma.</h2>
+                <ul class="features">
+                    <li><img width="127" height="127" src="img/icon-devices.svg" alt="electronic devices image" />
+                        <p class="caption">Internet for all devices</p>
+                    </li>
+                    <li><img width="127" height="127" src="img/icon-coffee.svg" alt="cup of coffee image" />
```

Comment:
In general, avoid using `width` and `height` attributes on your `img` tags. Handle image sizes directly in your CSS file.